### PR TITLE
Use getExpansion function in artifact diff operation

### PR DESCRIPTION
**Pull request by Chris0296**
https://alphora.atlassian.net/browse/APHL-866

### CODE REVIEW

File: plugin/cr/src/test/java/org/opencds/cqf/ruler/cr/r4/RepositoryServiceTest.java [hunk 1]
```diff
@@ -1581,10 +1581,11 @@ void artifact_diff_compare_executable() {
 		Parameters grouperChanges = returnedParams.getParameter().stream().filter(p -> p.getName().contains("/dxtc")).map(p-> (Parameters)p.getResource()).findFirst().get();
 		List<ParametersParameterComponent> deleteOperations = getOperationsByType(grouperChanges.getParameter(), "delete");
 		List<ParametersParameterComponent> insertOperations = getOperationsByType(grouperChanges.getParameter(), "insert");
+		// TODO these are now 0, is this expected with removal of dao for expansion?
```
Comment (TahaAttari) [inline plugin/cr/src/test/java/org/opencds/cqf/ruler/cr/r4/RepositoryServiceTest.java:1584]: hmm, no it should still be adding or removing the same number of codes. Before running the diff operation does the expanded ValueSet show the changes in expansion content?

File: plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java [hunk 2]
```diff
@@ -1261,12 +1262,8 @@ private void doesValueSetNeedExpansion(ValueSet vset, IFhirResourceDaoValueSet<V
 			ValueSetExpansionOptions options = new ValueSetExpansionOptions();
 			options.setIncludeHierarchy(true);
```
Comment (TahaAttari) [inline plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java:1263]: guess these aren't needed?

File: plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java [hunk 3]
```diff
@@ -2,15 +2,7 @@
...
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.TreeSet;
-import java.util.UUID;
+import java.util.*;
```
Comment (TahaAttari) [inline plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java:5]: nice

Comment (Chris0296) [reply plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java]: Fixed

Comment (TahaAttari) [reply plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java]: I wasn't asking for changes sorry, it did look cleaner 🙈

**Pull request merged**